compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -47,9 +47,8 @@ impl<'tcx> MirPass<'tcx> for Inline {
         match sess.mir_opt_level() {
             0 | 1 => false,
             2 => {
-                (sess.opts.optimize == OptLevel::Default
-                    || sess.opts.optimize == OptLevel::Aggressive)
-                    && sess.opts.incremental == None
+                sess.opts.optimize == OptLevel::Default
+                    || sess.opts.optimize == OptLevel::Aggressive
```

Comment:
Could this match on `optimize`? The names are kind of confusing, and seeing them ordered by strength could make me more comfortable.

Alternatively, `>= Default` (assuming `Default` is `-O` aka `-C opt-level=2`?) would be another way to indicate that it "starts" somewhere and increases with opt-level.